bwrobot/PointArray[] point_list
int32 num_models
